migration_to_v2_0_tests/tests/migrated_data_test.py

```diff
@@ -96,6 +102,29 @@ def test_layer_thumbnails(client, layer):
     assert diff_pixels <= DEFAULT_THUMBNAIL_PIXEL_DIFF_LIMIT, f"diff_pixels={diff_pixels}\nimg_path={img_path}\nexp_img_path={exp_img_path}"
 
 
+@pytest.mark.usefixtures("import_publication_uuids_fixture")
+@pytest.mark.parametrize("map", MAPS_TO_MIGRATE, ids=ids_fn)
+def test_maps_internal_layers(client, map: Map4Test):
+    query = f'''
+select ml.layer_uuid
+from {DB_SCHEMA}.publications map inner join
+     {DB_SCHEMA}.map_layer ml on map.id = ml.id_map
+where map.uuid = %s
+order by ml.id
+;
+'''
+    rows = db_util.run_query(query, data=(map.uuid,), uri_str=DB_URI)
+    exp_map_layer_uuids = {layer.uuid for layer in map.exp_internal_layers}
+    assert len(rows) == len(exp_map_layer_uuids), f"{rows=}, {exp_map_layer_uuids=}"
+    map_layer_uuids = {row[0] for row in rows}
+    assert map_layer_uuids == exp_map_layer_uuids, f"{rows=}, {exp_map_layer_uuids=}"
+
+    for layer in map.exp_internal_layers:
+        layer_info = client.get_workspace_publication(layer.type, layer.workspace, layer.name, actor_name=layer.owner)
+        layer_maps = layer_info['used_in_maps']
+        assert layer_maps == [{'workspace': map.workspace, 'name': map.name, }], f"{layer_maps=}, {map=}"
```

Comment:
Because this is test is parametrized by map, not by layer, I think it's more future-proof to assert that `{'workspace': map.workspace, 'name': map.name, }` is in `layer_maps` instead of testing it's the only item of `layer_maps`.

Reply:
Changed.